vm/src/vm.h

```diff
@@ -1,4 +1,5 @@
 #pragma once
+#include <cstdint>
```

Comment:
```suggestion
#include <stdint.h> // NOLINT(*-deprecated-headers)
```
This is a C compatible header file.